### PR TITLE
Create C03_11

### PR DESCRIPTION
**Pull request by DeedeeDL**
The program will create  a pattern like a right angle triangle with a number which will repeat a number in a row.

### CODE REVIEW

File: src/main/java/org/example/C03_11_PatternAngledTriangleRepeatedNumberInRow.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class C03_11_PatternAngledTriangleRepeatedNumberInRow {
```
Comment (andreiursudev) [inline src/main/java/org/example/C03_11_PatternAngledTriangleRepeatedNumberInRow.java]: "AngledTriangle" does not make much sense. Better "RightAngledTriangle"

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/C03_11_PatternAngledTriangleRepeatedNumberInRow.java (hunk 1 above)
Comment (DeedeeDL) [reply src/main/java/org/example/C03_11_PatternAngledTriangleRepeatedNumberInRow.java]: done

Comment (andreiursudev) [reply src/main/java/org/example/C03_11_PatternAngledTriangleRepeatedNumberInRow.java]: Now is worse. Give a name that describes the problem.

Comment (DeedeeDL) [reply src/main/java/org/example/C03_11_PatternAngledTriangleRepeatedNumberInRow.java]: Now?

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**